README.md

```diff
@@ -76,6 +76,92 @@ module "efs" {
+ * [`cloudwatch_exporter_cpu`]: String(optional): The cpu dedicated to cloudwatch_exporter. (default: 0)
+ * [`cloudwatch_exporter_memory`]: String(optional): The memory dedicated to cloudwatch_exporter. (default: 200)
+ * [`cloudwatch_exporter_memory_reservation`]: String(optional): The memory reservation to cloudwatch_exporter. (default: 100)
+ * [`grafana_cpu`]:String(optional): The cpu dedicated to grafana. (default: 0)
+ * [`grafana_memory`]:String(optional): The memory dedicated to grafana. (default: 100)
+ * [`grafana_memory_reservation`]:String(optional): The memory reservation to grafana. (default: 100)
+ * [`prometheus_version`]: String(optional): The version of the prometheus docker image . (default: v2.4.0)
+ * [`alertmanager_version`]: String(optional): The version of the alertmanager docker image . (default: v0.15.1)
+ * [`cloudwatch_exporter_version`]: String(optional): The version of the cloudwatch exporter docker image . (default: cloudwatch_exporter-0.5.0)
+ * [`grafana_version`]: String(optional): The version of the grafana's docker image . (default: latest)
+ * [`alb_arn`]: String(required): The arn of the alb.
+ * [`alb_sg_id`]: String(required): The security group of the alb.
+ * [`alb_ssl_cert`]:String(required): The certificate ARN of the alb.
+ * [`source_subnet_cidrs`]: List(required): The public CIDRs from where the prometheus monitoring is accessible.
+ * [`grafana_port`]: String(optional): The default grafana port the container is listening to. (default: 3000)
+ * [`opsgenie_api_key`]: String(required): The opsgenie api key for alerting.  
+ * [`slack_channel`]:  String(required): The slack channel where the alerts will be sent.  
+ * [`slack_url`]: String(required): The slack webhook used to access slack alerting.
+ * [`concourse_url`]: String(optional): Optional concourse URL to monitor. If not specified concourse won't be monitored. (default:"")
+ * [`cloudwatch_metrics`]: String(optional): Optional additional metrics scraping jobs. (default:"")
+ An example metric:
+ ```
+ <<EOF
+- aws_namespace: AWS/ELB
+ aws_metric_name: RequestCount
+ aws_dimensions: [AvailabilityZone, LoadBalancerName]
+ aws_dimension_select:
+   LoadBalancerName: [myLB]
+ aws_statistics: [Sum]
+EOF
+```
+* [`custom_jobs`]: String(optional): Optional additional prometheus scraping jobs (default:"")
+ * [`custom_alert_rules`]: String(optional): Optional additional alert rules for alertmanager (default:"")
+ * [`efs_subnets`]: List(required): The subnets where we want to deploy the required efs file system
+ * [`mount_point`]: String(optional): The mount point where we want to mount EFS in the ECS nodes. (default:"/monitoring")
+
+ ### Output
+ * [`alb_listener`]: The id of the ALB listener created for monitoring.
+ * [`efs_mount_point`]: The mount point where we want to mount EFS in the ECS nodes.
+ * [`efs_dns_name`]: The EFS DNS name
+
+### Example
+```
+module "monitoring" {
```

Comment:
Can you change the example with direct (example) values instead of pointing to data sources? Makes it easier for people that don't have our stack structure to understand how the module works.

I've recently changed the examples of some of our open-source modules for this reason -> https://github.com/skyscrapers/terraform-sqs/commit/63a6b6c033e9f1fa3bba96648dcdda73b60fc664#diff-04c6e90faac2675aa89e2176d2eec7d8L48